htdocs/admin/tools/ui/components/test_arrays.php

```diff
@@ -1,25 +1,47 @@
 <?php
 /* Copyright (C) 2024       Frédéric France         <frederic.france@free.fr>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
  */
 
 
 // Load Dolibarr environment
-@include '../../main.inc.php';
+require '../../../../main.inc.php';
 
-// Security
-if ($dolibarr_main_prod) {
-	accessforbidden('Access forbidden when $dolibarr_main_prod is set to 1');
```

Comment:
Why removing this ? As it is not a fearure for end users, but for dev only, this protection avoid any vulnerability in this feature ok n priduction.

Reply:
oh sorry, too large copy paste